elpis/engines/common/input/clean_json.py

```diff
@@ -198,13 +220,15 @@ def deal_with_punctuation(text: str = '',
     :param punctuation_to_explode_by: punctuation marks to replace with spaces
     :return: cleaned text
     """
-    pattern_to_collapse_by = re.escape(punctuation_to_collapse_by)
-    pattern_to_explode_by = re.escape(punctuation_to_explode_by)
-    # Prioritise exploding first, this is set of punctuation marks that the user adds
-    new_text: str = re.sub(rf"[{pattern_to_explode_by}]", " ", text)
+    new_text: str = text
+    # Prioritise exploding first, these are punctuation marks that the user sets
+    if punctuation_to_explode_by is not '':
```

Comment:
This should be `!=` not `is not`. Same for below.

Might also be worth checking for `is not None` but up to you.

Reply:
ok